django-stubs/db/models/base.pyi

```diff
@@ -67,7 +67,7 @@ class Model(metaclass=ModelBase):
         self, using: Any = ..., keep_parents: bool = ...
     ) -> Tuple[int, Dict[str, int]]: ...
     def full_clean(
-        self, exclude: Optional[Collection[str]] = ..., validate_unique: bool = ...
+        self, exclude: Optional[Collection[str]] = ..., validate_unique: bool = True, validate_constraints: bool = True ...
```

Comment:
```suggestion
        self, exclude: Optional[Collection[str]] = ..., validate_unique: bool = True, validate_constraints: bool = True
```

but I'm wondering if we want to keep the default values, I think type stubs are not supposed to have values defined as defaults, instead they're supposed to use `...`

Reply:
I agree - but I had to add them for the linter to pass?